docs/Authentication.md

```diff
@@ -104,6 +104,12 @@ const authProvider = {
                 throw new Error('Network error')
             });
     },
+    checkAuth: () => {
+        return localStorage.getItem('auth') ? Promise.resolve() : Promise.reject();
```

Comment:
The problem is that the Authentication chapter mentions that method later ('checking credentials during navigation'). So basically we're adding code without explaining it here, and we're re-adding it later. It doesn't really make sense. 

I'd prefer that you add resolved promises for both, with a comment explaining that these methods are required for the authentication system to work.